DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -24,10 +24,16 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
         public CourseContent? GetCourseContent(int customisationId)
         {
             return connection.QueryFirstOrDefault<CourseContent>(
-                @"SELECT C.CustomisationID AS Id, C.CustomisationName, A.ApplicationName 
-                      FROM Customisations as C
-                      JOIN Applications AS A ON C.ApplicationID = A.ApplicationID
-                      WHERE C.CustomisationID = @customisationId",
+                @"SELECT Customisations.CustomisationID AS id,
+                         Applications.ApplicationName,
+	                     Customisations.CustomisationName,
+	                     dbo.GetMinsForCustomisation(Customisations.CustomisationID) AS AverageDuration,
+	                     Centres.CentreName,
+	                     Centres.BannerText
+                  FROM Applications
+                  INNER JOIN Customisations ON Applications.ApplicationID = Customisations.ApplicationID
+                  INNER JOIN Centres ON Customisations.CentreID = Centres.CentreID
+                  WHERE Customisations.CustomisationID = @customisationId;",

```

Comment:
Could you also make this return the progress id? Found in the progress table by searching with the customisation id and candidate id (will need to be passed in). This will be needed later to fetch section info, and will be needed by Frank I've just realised 🙂

Reply:
Actually looking at some of the upcoming tickets I think this might make more sense as a separate method (because we might need to create the progress record if it doesn't exist). Sorry about the confusion @benjimarshall!

Reply:
@kwunkiuma if you create a service method to get the progress id (which will find it from progress table based on customisation id and candidate id), in addition to your update service method. Then you can call that in the controller to get the progress id, before calling your update method. Does that make sense? And do you think it fits with what you're doing @benjimarshall? 🙂

Reply:
Yes, I should be able to do that.

Reply:
I think I follow what's going on here 🙂 Just to confirm, there's nothing with progressIDs that you want added to this branch?

Reply:
No, nothing needed on this branch 🙂